build.proj

```diff
@@ -52,7 +52,8 @@
 
   <Target Name="Build" DependsOnTargets="$(BuildDependsOn)" />
   <Target Name="Clean">
-    <RemoveDir Directories="$(BinDir)" />
+    <RemoveDir Directories="$(ArtifactsObjDir)" />
+    <RemoveDir Directories="$(ArtifactsBinDir)" />
```

Comment:
What about additional directories in the artifacts dir? Cleaning the artifacts dir itself is handled by whom?

Reply:
I just copied this from https://github.com/dotnet/corefx/blob/master/build.proj#L70-L74, but you're right, I'll change this to remove `ArtifactsDir`

Reply:
I think the idea behind that was that the other stuff contains the restored tools (similar to our old tools folder) so folks might not want to always delete that on a clean.  Just a guess.

Reply:
Whatever it was, I had to revert back to what it was, the build broke in SourceLink on Unix when I cleaned `ArtifactsDir`